dags/atd_service_bot_intake.py

```diff
@@ -0,0 +1,78 @@
+import os
+from datetime import datetime, timedelta
+
+
+from airflow.models import DAG
+from airflow.operators.docker_operator import DockerOperator
+
+from onepasswordconnectsdk.client import Client, new_client
+import onepasswordconnectsdk
+
+DEPLOYMENT_ENVIRONMENT = os.getenv("ENVIRONMENT", "development")
+ONEPASSWORD_CONNECT_HOST = os.getenv("OP_CONNECT")
+ONEPASSWORD_CONNECT_TOKEN = os.getenv("OP_API_TOKEN")
+VAULT_ID = os.getenv("OP_VAULT_ID")
+
+default_args = {
+    "owner": "airflow",
+    "description": "Fetch new DTS service requests and create Github issues",
+    "depends_on_past": False,
+    "start_date": datetime(2015, 12, 1),
+    "email_on_failure": False,
+    "email_on_retry": False,
+    "retries": 0,
+}
+
+docker_image = "atddocker/atd-service-bot:production"
+
+REQUIRED_SECRETS = {
+    "KNACK_APP_ID": {
+        "opitem": "Service Bot",
```

Comment:
This is a great point, and something that Frank and I have touched on here and there. Finding good ways to prevent duplicate items tucked away. I could see us getting into a jam where a password updates, and we only update one item for one ETL because we forgot about another one elsewhere in this vault.

I'm taking notes on all these good points that you are bring up, and I'm going to float the idea of a near future Airflow 2 meeting today in dev sync. That way we can come away from with decisions on things and get some of the ETLs running. ⚙️